docs/app/Examples/collections/Form/Variations/FormExampleInverted.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Button, Form, Segment } from 'semantic-ui-react'
+
+
+const FormExampleInverted = () => (
+  <Segment inverted>
+    <Form inverted>
+      <Form.Group widths='equal'>
+        <Form.Field label='First name' control='input' placeholder='First name' />
+        <Form.Field label='Last name' control='input' placeholder='Last name' />
```

Comment:
Apologies that I missed this on the first pass.  Let's use the shorthand `<Form.Input />` for brevity.  I like to keep the doc examples as slim as possible:

```diff
-<Form.Field label='First name' control='input' placeholder='First name' />
+<Form.Input label='First name' placeholder='First name' />
```